src/main.js

```diff
@@ -112,7 +112,8 @@ var currentPoster = {
 //currentPoster = new Poster('', '', '')
 
 // event listeners go here 👇
-window.addEventListener('load', makeRandomPoster) {
+// window.addEventListener('load', makeRandomPoster);
+window.onload(makeRandomPoster);
```

Comment:
Great work!

Reply:
Looks like we both missed that we did not uncomment line 115 and then remove 116.

Reply:
We will resolve this on the next push.